src/components/modals/StepConnectDevice.js

```diff
@@ -34,3 +35,10 @@ export default (props: Props) => (
     )}
   />
 )
+
+StepConnectDevice.defaultProps = {
+  account: undefined,
+  currency: undefined,
```

Comment:
shouldn't define these because it's already the default.
if you do define these, then the prop is actually a 

```
account: ?Account
currency: ?Currency
```

if you don't, it's a 

```
account?: Account
currency?: Currency
```

which allows to not pass-in props and don't define this

Reply:
Thanks for the tip !